assets/js/modules/analytics/datastore/accounts.js

```diff
@@ -208,6 +212,31 @@ export const selectors = {
 
 		return error || null;
 	},
+
+	/**
+	 * Checks whether an Analytics account should be created or not.
+	 *
+	 * @since n.e.x.t
+	 * @private
+	 *
+	 * @return {boolean} Whether an Analytics account should be created or not.
+	 */
+	isCreateAccount: createRegistrySelector( ( select ) => () => {
+		return ACCOUNT_CREATE === select( STORE_NAME ).getAccountID();
+	} ),
```

Comment:
I'd prefer to get rid of this selector and do this comparison in the component where needed, that should be simple enough. Components already need to have awareness of these constants, so I think having this selector doesn't provide much benefit.